unlock-app/src/components/interface/checkout/CheckoutWrapper.tsx

```diff
@@ -48,8 +48,8 @@ const CloseButton = styled(Close).attrs(() => ({
   className: 'closeButton',
 }))`
   position: absolute;
-  top: 24px;
-  right: 24px;
+  top: 12px;
+  right: 12px;
```

Comment:
Moved this so it wouldn't come beyond the padding and clash with child components